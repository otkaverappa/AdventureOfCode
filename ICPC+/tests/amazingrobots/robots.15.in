20 20
####################
#X#...#...#...#....#
#.#.#.#.#.#.#.#.#.##
#.#.#.#.#.#.#.#.#..#
#.#.#.#.#.#.#.#.##.#
#.#.#.#.#.#.#.#.#..#
#.#.#.#.#.#.#.#.#.##
#.#.#.#.#.#.#.#.#..#
#.#.#.#.#.#.#.#.##.#
#.#.#.#.#.#.#.#.#..#
#.#.#.#.#.#.#.#.#.##
#.#.#.#.#.#.#.#.#..#
#.#.#.#.#.#.#.#.##.#
#.#.#.#.#.#.#.#.#..#
#.#.#.#.#.#.#.#.#.##
#.#.#.#.#.#.#.#.#..#
#.#.#.#.#.#.#.#.#..#
#.#.#.#.#.#.#.#.#..#
#...#...#...#...#.##
#################.##
1
18 18 2 N
20 20
####################
#..................#
...................#
#..................#
#..................#
#..................#
#..................#
#..................#
#..................#
#..................#
#..................#
#..................#
#..................#
#..................#
#..................#
#..................#
#..................#
#..................#
#.................X#
####################
4
5 2 4 N
6 2 4 N
7 2 4 N
2 3 3 S
