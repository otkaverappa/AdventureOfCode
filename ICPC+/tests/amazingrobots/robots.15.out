200
E
S
S
S
S
S
S
S
S
S
S
S
S
S
S
S
S
S
E
E
N
N
N
N
N
N
N
N
N
N
N
N
N
N
N
N
N
E
E
S
S
S
S
S
S
S
S
S
S
S
S
S
S
S
S
S
E
E
N
N
N
N
N
N
N
N
N
N
N
N
N
N
N
N
N
E
E
S
W
W
W
W
W
W
W
W
W
W
W
W
W
W
W
W
W
W
W
E
S
S
S
S
S
S
S
S
S
S
S
S
S
S
S
S
E
E
N
N
N
N
N
N
N
N
N
N
N
N
N
N
N
N
N
E
E
S
S
S
S
S
S
S
S
S
S
S
S
S
S
S
S
S
E
E
N
N
N
N
N
N
N
N
N
N
N
N
N
N
N
N
N
E
E
S
S
E
S
S
W
S
S
E
S
S
W
S
S
E
S
S
W
S
S
E
S
S
W
S
S
S
